src/KafkaFlow.OpenTelemetry/ExtensionMethods.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace KafkaFlow.Configuration
+{
+    using KafkaFlow.OpenTelemetry;
+
+    /// <summary>
+    /// Adds OpenTelemetry instrumentation
+    /// </summary>
+    public static class ExtensionMethods
+    {
+        /// <summary>
+        /// Adds OpenTelemetry instrumentation
+        /// </summary>
+        /// <param name="builder">The Kafka configuration builder</param>
+        /// <returns></returns>
+        public static IKafkaConfigurationBuilder AddOpenTelemetryInstrumentation(this IKafkaConfigurationBuilder builder)
+        {
+            builder.SubscribeGlobalEvents(hub =>
+            {
+                hub.MessageConsumeStarted.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryConsumerEventsHandler.OnConsumeStarted(eventContext.MessageContext);
+                });
+
+                hub.MessageConsumeError.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryConsumerEventsHandler.OnConsumeError(eventContext.MessageContext, eventContext.Exception);
+                });
+
+                hub.MessageConsumeCompleted.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryConsumerEventsHandler.OnConsumeCompleted(eventContext.MessageContext);
+                });
+
+                hub.MessageProduceStarted.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryProducerEventsHandler.OnProducerStarted(eventContext.MessageContext);
+                });
+
+                hub.MessageProduceError.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryProducerEventsHandler.OnProducerError(eventContext.MessageContext, eventContext.Exception);
+                });
+
+                hub.MessageProduceCompleted.Subscribe(eventContext =>
+                {
+                    return OpenTelemetryProducerEventsHandler.OnProducerCompleted(eventContext.MessageContext);
+                });
```

Comment:
Could you simplify this and use a shorter syntax using `=>`?

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5